### ISSUE TITLE
There is no jump straight to where the function defined

### ISSUE THREAD

**Issue by mqliutie** [SublimeText/CTags#289]
The directory structure is as follows

![test](http://images.cnblogs.com/cnblogs_com/mqliutie/687995/o_test.png)

The result is as follows

![result](http://images.cnblogs.com/cnblogs_com/mqliutie/687995/o_result.png)

How can I solve this problem？ Thank you

**Comment by stephenfin** [comment]
tl;dr: If I have understood your issue then the answer is: CTags (and therefore SublimeCTags) can't do this.

Long version: I'm not 100% sure what your issue is. If I understand you correctly, you're asking why SublimeCTags isn't intelligent enough to figure out which Javascript file to open (it's clearly `test2/index.js`)? If this is the case, then I'm afraid to tell you that this isn't an issue with SublimeCTags but rather with CTags itself. CTags isn't clever enough to do this as it works by using regexes and line numbers to find matching keywords, but does not attach any semantics to this. If you want this kind of clever matching you'll need to use a real IDE with a feature like IntelliSense built in.
